portmap/core/models.py

```diff
@@ -88,3 +88,7 @@ class QueryLog(BaseModel):
     datatype = models.CharField(max_length=30)
     source = models.TextField(max_length=100)
     destination = models.TextField(max_length=100)
+
+class DataType(BaseModel):
+    name = models.CharField(max_length=30)
```

Comment:
It seems fine for the moment for the articles table NOT to have a foreign key to this one.  WE can always change that in a migration later if we want the tables to link to each other and ensure integrity of the data type names.  So I agree with this approach at least util we find need to enforce the dependency or follow the foreignkey relationship.